modules/odc-grouping-multiple-components.adoc

```diff
@@ -15,23 +15,23 @@ The following procedure adds a MongoDB database service to an existing applicati
 
 .Procedure
 
-. Create and deploy the MongoDB service to your project as follows:
-.. In the *Developer* perspective, navigate to the *Add* view and select the *Database* option to see the *Developer Catalog*, which has multiple options that you can add as components or services to your application.
-.. Click on the *MongoDB* option to see the details for the service.
-.. Click *Instantiate Template* to see an automatically populated template with details for the MongoDB service, and click *Create* to create the service.
+. Create and deploy the MariaDB service to your project as follows:
+.. In the *Developer* perspective, navigate to the *+Add* view and select the *Database* option to see the *Developer Catalog*, which has multiple options that you can add as components or services to your application.
+.. Click on the *MariaDB* option to see the details for the service.
+.. Click *Instantiate Template* to see an automatically populated template with details for the MariaDB service, and click *Create* to create the service.
 
-. On the left navigation panel, click *Topology* to see the MongoDB service deployed in your project.
-. To add the MongoDB service to the existing application group, select the *mongodb* pod and drag it to the application; the MongoDB service is added to the existing application group.
-. Dragging a component and adding it to an application group automatically adds the required labels to the component. Click on the MongoDB service node to see the label `app.kubernetes.io/part-of=myapp` added to the *Labels* section in the *Overview* Panel.
+. On the left navigation panel, click *Topology* to see the MariaDB service deployed in your project.
```

Comment:
User will automatically redirected to Topology page and doesn't need to click anywhere.

Reply:
Yes